thirdparty/tesseract/no_debug_fonts.patch

```diff
@@ -0,0 +1,11 @@
+--- i/src/ccstruct/debugpixa.h
++++ w/src/ccstruct/debugpixa.h
+@@ -14,7 +14,7 @@
+   // TODO(rays) add another constructor with size control.
+   DebugPixa() {
+     pixa_ = pixaCreate(0);
+-#ifdef TESSERACT_DISABLE_DEBUG_FONTS
++#if 1//def TESSERACT_DISABLE_DEBUG_FONTS
```

Comment:
Patching feels a bit convoluted compared to just setting it?

Reply:
There's no knob in the build system to enable the flag… And it's only used here, so it's actually easier then having to patch `CMakeLists.txt`, or create a dedicated toolchain file to tweak the compilation flags just for tesseract.

(I almost used something like `set(CMAKE_C_FLAGS_INIT "$(strip $(CFLAGS)) $${CPPFLAGS} $${CFLAGS}")` when generating the toolchain file to make it easier to tweak those).

Reply:
Strange, but alright.